src/domelts.js

```diff
@@ -0,0 +1,72 @@
+const getDate = () => {
+  const currentDate = document.querySelector('#date');
+  currentDate.textContent = new Date().toUTCString();
+};
+
+async function getWeather(value = 'Bamenda') {
+  const response = await fetch(`https://api.openweathermap.org/data/2.5/weather?q=${value}&APPID=a65bf913c63cc981687c4ea80470725d`, { mode: 'cors' });
+  const weatherdata = await response.json();
+
+  const [weather] = weatherdata.weather;
+  const { main } = weather;
+  const { temp, feelLike, humidity } = weatherdata.main;
+  const { speed } = weatherdata.wind;
+  const { country } = weatherdata.sys;
+  const { name } = weatherdata;
+
+  const ctryName = document.querySelector('#ctry');
+  ctryName.textContent = `${name},  ${country}`;
+
+  const tempCelcius = (temp - 273).toFixed(2);
+  const tempFahrenheit = ((tempCelcius * 1.8000) + 32).toFixed(2);
+
+  const feelsCelcius = (feelLike - 273).toFixed(2);
+  const feelsFahrenheit = ((feelsCelcius * 1.8000) + 32).toFixed(2);
```

Comment:
- [ ] The app should make a new request to change the temperature between Celsius and Fahrenheit.
![Screenshot from 2020-04-20 21-16-56](https://user-images.githubusercontent.com/30977845/79811464-4bc24f00-834c-11ea-8bbb-dd554d61729a.png)